hera_mc/geo_handling.py

```diff
@@ -158,68 +153,60 @@ def get_station_types(self, add_stations=True):
         for key in self.station_types.keys():
             self.flipped_station_types[self.station_types[key]['Name']] = key
 
-    def is_in_geo_location(self, station_name):
+    def is_in_database(self, station_name, db_name='geo_location'):
         """
-        checks to see if a station_name is in the geo_location database
+        checks to see if a station_name is in the named database
 
         return True/False
 
         Parameters:
         ------------
         station_name:  string name of station
+        db_name:  name of database table
         """
         ustn = station_name.upper()
-        station = self.session.query(geo_location.GeoLocation).filter(
-            func.upper(geo_location.GeoLocation.station_name) == ustn)
+        if db_name == 'geo_location':
+            station = self.session.query(geo_location.GeoLocation).filter(
+                func.upper(geo_location.GeoLocation.station_name) == ustn)
+        elif db_name == 'connections':
+            station = self.session.query(part_connect.Connections).filter(
+                func.upper(part_connect.Connections.upstream_part) == ustn)
+        else:
+            raiseValueError('db not found.')
         if station.count() > 0:
             station_present = True
         else:
             station_present = False
         return station_present
 
-    def is_in_connections(self, station_name, active_date=None, return_antrev=True):
+    def find_antenna_at_station(self, station, query_date):
         """
-        checks to see if the station_name is in the connections database
-            (which means it is also in parts)
-        if active_date is None, it will return True/False if ever connected
-        if active_date is given, it will check whether it was connected at that time
+        checks to see what antenna is at a station
 
-        return True/False unless (1) active_date is provided and (2) return_antrev is True
+        Returns None or the active antenna_name and revision (must be an active
```

Comment:
I would like if the docstring was a bit more explicit about the format of the return value; something like

> Returns a tuple `(antenna_name, antenna_revision)`, representing the antenna that was active at the date `query_date`, or None if no antenna was active at the station. Raises `ValueError` if the database lists multiple active connections at the station at `query_date`.